client/app/dedicatedCloud/datacenter/drp/steps/ovh/final/dedicatedCloud-datacenter-drp-ovh-finalStep.routes.js

```diff
@@ -0,0 +1,14 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.datacenter.drp.ovh.finalStep', {
+    url: '',
+    views: {
+      'stepView@app.dedicatedClouds.datacenter.drp': {
```

Comment:
```suggestion
      'stepView@app.dedicatedClouds.datacenter.drp': 'dedicatedCloudDatacenterDrpOvhFinalStep'
```
You use this name more than once, it should be a constant